lib/js/styles/ColorsTokensBodywork/ColorsTokensBodywork.stories.mdx

```diff
@@ -0,0 +1,30 @@
+import { Meta, Story, Props } from '@storybook/addon-docs';
+import ColorsTokensBodywork from './ColorsTokensBodywork';
+import tokensBodywork from '../../../styles/settings/colors/_tokens-bodywork.json';
+
+<Meta title="foundations/Tokens/ColorsBodywork" component={ColorsTokensBodywork} />
+
+# Card
+
+<Story
+	name="ColorsBodywork"
+	args={{
+		tokensBodywork: tokensBodywork,
+	}}
+	argTypes={{
+		tokensBodywork: Object,
+	}}
+>
+	{(args) => {
+		return {
+			components: { ColorsTokensBodywork },
+			template:
+				'<colors-tokens-bodywork :tokensBodywork="tokensBodywork"></colors-tokens-bodywork>',
```

Comment:
Instead of using separate component for theme you can add class to existing component:
`template: '<ColorsList class="theme-bodywork" :colorsLists="tokensBodywork" />',`

Reply:
I made in in different way, because of one view where we display many themes: wnl, bodywork - there class need be added on the singiel list level.